src/domain/enrolment/enrolmentForm/ValidationSchema.tsx

```diff
@@ -74,6 +83,15 @@ export default Yup.object().shape({
               }));
             }
           ),
+        extraNeeds: Yup.string().when([], () => {
+          /* 
+          A little Yup-hack with an empty when-block, 
+          since the isMandatoryAdditionalInformationRequired is not part of studyGroup,
+          it needs to be fetched from 1st level when-block so it would be populated.
+          */
+          if (!isMandatoryAdditionalInformationRequired) return Yup.string();
+          return Yup.string().required(VALIDATION_MESSAGE_KEYS.STRING_REQUIRED);
+        }),
```

Comment:
Would the following work?

```
  extraNeeds: isMandatoryAdditionalInformationRequired
          ? Yup.string().required(VALIDATION_MESSAGE_KEYS.STRING_REQUIRED)
          : Yup.string(),
```

Reply:
I need to try that. Couldn't think so simple :D

Reply:
it works!